### ISSUE TITLE
MV2 list not working anymore since the last update

### ISSUE THREAD

**Issue by Gandalf-The-Wise** [Kees1958/W3C_annual_most_used_survey_blocklist#81]
I get 0 of 0 when I load the the combined MV2 version in uBlock Origin.
(https://raw.githubusercontent.com/Kees1958/W3C_annual_most_used_survey_blocklist/master/EU_US_MV2_most_common_ad%2Btracking_networks.txt)

**Comment by yokoffing** [comment]
https://github.com/Kees1958/W3C_annual_most_used_survey_blocklist/pull/83

**Comment by Kees1958** [comment]
Sorry corrected (Wrong include notation)

**Comment by Kees1958** [comment]
> I get 0 of 0 when I load the the combined MV2 version in uBlock Origin. (https://raw.githubusercontent.com/Kees1958/W3C_annual_most_used_survey_blocklist/master/EU_US_MV2_most_common_ad%2Btracking_networks.txt)

Thanks for reporting. Sorry for the late reaction (I was on holiday in Egypt). Everything should be okat now. I wanted to drop a reply to thank you. I stopped browsing security.nl and just see which news you post on MT and WSF (so you are my Gandalf teh security news feed :-)

**Comment by Kees1958** [comment]
@Gandalf-The-Wise 

My work only allows ABP or Ghostery exyension in adblocker category (probably because AG is Russian and ABP and Ghistery are made by German developers). Because I like to keep settings the same on private and work laptop I moved away from AG and I first tried ABP but switched to Ghostery because of a new feature they are testing: https://github.com/ghostery/ghostery-extension/issues/1475

It made it to production, but it is still hidden. You need to click 5 times on the version number (at the bottom of settings page). I copied the Dutch news websites from my 'personal ABP blocklist'  into the 'custom filters'  section and it work flawlessly. Only small glitch is that comment lines (starting with !) are not recognized, so you need to remove them.

**Comment by Gandalf-The-Wise** [comment]
@Kees1958 
It makes sense to keep it the as much same as possible across work and home setups.
At the moment my work allows uBlock Origin (not the Lite version), Adblock Plus and Ghostery.
But that will possibly change after summer because my department was sold off.
I tried both ABP and Ghostery, but I am disappointed in ABP and Youtube ads and the adblocking part of Ghostery in general (an example of that is tweakers.net).
I rate AdGuard > uBlock Origin > Adblock Plus > Ghostery in terms quality of adblocking on the sites I visit daily.
I am still not sure what to use on Edge when MV3 will be a reality.

**Comment by Kees1958** [comment]
@Gandalf-The-Wise 

Organizational changes allways have impact, Hopefully for you for the better.

Have you tried Ghostery recently? It does better than ABP and similar to uBO-light. With recent custom rules it works well for me. 

They also respond quick on website breakage reports.

**Comment by Gandalf-The-Wise** [comment]
@Kees1958 
Thanks, for now, it looks good, I see no negatives with this takeover at the moment.

Currently I am using uBO Lite, I will look into Ghostery again when I have more time.
I find its settings less clear than with other adblockers.
And thanks, for the custom filters.

**Comment by yokoffing** [comment]
> @Kees1958 I reported tweakers.net at Ghostery: `ghostery/broken-page-reports#719`

You're better off reporting this to https://github.com/easylist/easylist/issues, as they're just sourcing rules from the same filters as uBO defaults (more or less). I get ads on the page here, too,  in the USA without any regional filters.

**Comment by Kees1958** [comment]
@yokoffing Thanks Hal

I will have a look at the Dutch EL, to see whether those 2 first party rules are in.

When I understand you correctly Ghostery uses EL for ads and its own tracker DB for privacy?

**Comment by yokoffing** [comment]
> @Kees1958 so Ghostery is using only English filters for ads

Ghostery's MV2 extension detected and used regional filters after a certain period of time. The MV3 extension will support regional filters in the future, according to [their roadmap](https://www.ghostery.com/blog/launching-ghostery-10-adblocker#is-ghostery-10-worth-it).

> and is using its own tracker DataBase for privacy

Ghostery protects users from web tracking in [two main ways](https://www.ghostery.com/blog/how-we-at-ghostery-protect-users-from-web-tracking#protecting-our-users-from-web-tracking):

* **Filter lists:** Ghostery uses uBlock Origin's default lists, uBlock filters - Annoyances list, and EasyList Cookie.
  * Last I checked, Ghostery was still attempting to support all of uBlock Origin's scriplets (which, honestly, is what makes uBO so good).
* **Heuristics:** Ghostery employs its [TrackerDB](https://www.ghostery.com/blog/introducing-trackerdb) for "anti-tracking technology" to [selectively block](https://www.ghostery.com/blog/how-ghostery-categorizes-trackers#selective-blocking) certain tracker categories.

In other words, filter lists block most unwanted content, while TrackerDB sifts the rest for [unique identifiers](https://www.ghostery.com/blog/how-we-at-ghostery-protect-users-from-web-tracking#ghostery-anti-tracking) (UIDs).

**Comment by Kees1958** [comment]
@yokoffing Thanks fo rthe explanation

Ghostery mentions it will facilitate most (80-90%) of uBO and AG filter options. For people like me which can;t use Russian extension on work PC (enforced bu GPO), Ghostery is the best alternative to uBO Mv2 and AG Mv3 (they will also support scriptlests in custom rules in the future). I tried ABP, but Ghostery does a better job on tough websites like youtube.
